src/thoughts-on/ariadnes-thread.html

```diff
@@ -0,0 +1,317 @@
+              <li>Go back in the thread to a previous snapshot</li>
+              <li>Review the interaction to date (thread history)</li>
+              <li>Break down unwieldy interactions into manageable parts</li>
+              <li>
+                Penetrate further into the labyrinth without fear of pitfalls
+              </li>
+            </ul>
+          </section>
+          <aside>
+            <p>
+              See the Ariadne's Thread project repo
+              <a
+                href="https://github.com/p-n-c/ariadnes-thread/blob/main/README.md"
+                >README</a
+              >
+              for hints and tips.
+            </p>
+          </aside>
+          <section aria-labelledby="Conclusion">
+            <h2 id="Conclusion">Conclusion</h2>
+            <p>
+              LLMs are excellent at small, discrete tasks. They are typically
+              reliable at convergent thinking (refining ideas) and with a little
+              oversight can help speed things along.
+            </p>
+            <p>
+              They also expand our ambitions by plugging gaps (handling
+              unfamiliar languages or APIs) and doing heavy lifting (writing
+              tests, generating diagrams).
+            </p>
+            <p>
+              They are endlessly patient but they are not cheap. Their cost is
+              paid in finite resources such as water, pollution and emissions as
+              well as money.
+            </p>
+            <section aria-labelledby="ariadnes-thread">
+              <h3 id="ariadnes-thread">Ariadne's Thread</h3>
+              <p>
+                Ariadne's Thread emerged from two different but complementary
+                wishes.
+              </p>
+              <ul>
+                <li>To be independent of LLMs (because of their cost)</li>
+                <li>
+                  To know where our interactions are taking us (so as to not get
+                  lost)
+                </li>
+              </ul>
+            </section>
+            <section aria-labelledby="dependency-avoidance">
+              <h4 id="dependency-avoidance">Dependency avoidance</h4>
+              <p>
+                We use LLMs (quite a lot) but we are not reliant on them. Where
+                possible we create something new from our interactions,
+                something which does not require their continued presence. We
+                think LLMs are valuable but not indispensable.
+              </p>
+              <p>
+                The work on Ariadne's Thread is itself an example of this
+                principle in practice; we worked with three LLMs to test its
+                viability but we haven't made the solution dependent on having
+                access to an LLM.
+              </p>
+            </section>
+            <section aria-labelledby="critical-thinking">
+              <h4 id="critical-thinking">Critical thinking</h4>
+              <p>
```

Comment:
This is a really good section, I think you could expand on it. It's central to your philosophy and to the thread. I would even put this earlier in the article.